### DISCUSSION TITLE
서버의 저장값을 제거하는 기능이 필요합니다

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#841] [4. General]
1. 월드 파일 등록시 3단계에서 실패시 월드를 포기해야하는 상황발생
2. 진행도중 저장값에 오류가 생긴 계정은 다시는 플레이 불가능한 계정이되는 상황발생

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 일반적으로는 업로드 실패시 재업로드가 가능한 상태로 돌아오는데 아예 재업로드가 불가능해 진 걸까요?
관련해서 스크린샷을 부탁 드립니다.
